compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -372,6 +373,59 @@ object SymDenotations {
       case Nil => Nil
     }
 
+    /** If this is a method, the parameter symbols, by section.
+     *  Both type and value parameters are included. Empty sections are skipped.
+     */
+    final def rawParamss: List[List[Symbol]] = myParamss
+    final def rawParamss_=(pss: List[List[Symbol]]): Unit =
+      myParamss = pss
+
+    final def setParamss(tparams: List[Symbol], vparamss: List[List[Symbol]])(using Context): Unit =
+      rawParamss = (if tparams.isEmpty then vparamss else tparams :: vparamss)
+        .filterConserve(!_.isEmpty)
+
+    final def setParamssFromDefs(tparams: List[TypeDef[?]], vparamss: List[List[ValDef[?]]])(using Context): Unit =
+      setParamss(tparams.map(_.symbol), vparamss.map(_.map(_.symbol)))
+
+    /** A pair consistsing of type paremeter symbols and value parameter symbol lists
```

Comment:
@nicolasstucki The `paramSymss ` method should be exported through Tasty reflect.

Reply:
I added `paramSymss` to the TASTy reflect API